src/constants.ts

```diff
@@ -7,4 +7,4 @@ export const SETTINGS_FILE_PATH_ID = "bibtex.file";
 export const ADD_BIBTEX_REFERENCE_COMMAND = "addBibTeXReference";
 export const PLUGIN_TOOLBAR_BUTTON_ID = "bibtex_plugin_toolbar_button";
 export const CITATION_POPUP_ID = "citation_popup";
-export const ERROR_PARSING_FAILED = "Error= Something wrong happened while parsing";
+export const ERROR_PARSING_FAILED = "Error while parsing: ";
```

Comment:
"Error while parsing BibTex file:", so that users know it's about their file.

Reply:
Got it